python/cuml/tests/test_random_forest.py

```diff
@@ -384,7 +384,10 @@ def test_rf_classification_unorder(
         sk_model.fit(X_train, y_train)
         sk_preds = sk_model.predict(X_test)
         sk_acc = accuracy_score(y_test, sk_preds)
-        assert acc >= (sk_acc - 0.07)
+        # Increased tolerance to 0.10 to account for RNG variance from bias fix
```

Comment:
Nit: we don't need a comment about the change, but a comment describing how the tolerance was arrived on is useful. Can just drop this line

```suggestion
```